lib/saviour/base_uploader.rb

```diff
@@ -64,6 +64,10 @@ def store_dir
       @store_dir ||= Uploader::StoreDirExtractor.new(self).store_dir
     end
 
+    def storage
```

Comment:
no need for this no? it's only a class attribute

Reply:
it is easier from api to call it `uploader.storage` than `uploader.class.storage`, for example in `LifeCycle`

Reply:
well, I see it's used, then I would just remove the memoization here. It's already memoized in the class.